### ISSUE TITLE
does agora supports unity UWP/Hololens build

### ISSUE THREAD

**Issue by AnandMulay** [AgoraIO/Basic-Video-Broadcasting#82]
as per title agora video streaming or broadcast support for hololens, does it work on this platform ?

**Comment by plutoless** [comment]
we don't have ready-to-go sdk but you can wrap the c++ api yourselves on UWP.

**Comment by mitra7** [comment]
Do you have any plans to support this kind of platform? I'm also interested on this topic.

**Comment by zhangtao1104** [comment]
Yes, we plan to support UWP, it is in our roadmap. We maybe support UWP in q2 of next year.
